cachito/web/static/api_v1.yaml

```diff
@@ -192,7 +192,9 @@ paths:
                   items:
                     type: array
                     items:
-                      $ref: "#/components/schemas/Request"
+                      oneOf:
+                        - $ref: "#/components/schemas/Request"
+                        - $ref: "#/components/schemas/RequestVerbose"
```

Comment:
> the array will be of all either Request or VerboseRequest

Not sure if my intuition is correct, but doesn't the current schema essentially mean
```python
list[Request | RequestVerbose]
```
When what we want is
```python
list[Request] | list[RequestVerbose]
```
Maybe the oneOf should go around the whole array, rather than the items of this array?

Reply:
Good point! The validation fails using oneOf when returning an empty array (since it matches both schemas), but using anyOf appears to work. I'll update